packages/mdc-ripple/_mixins.scss

```diff
@@ -141,21 +199,57 @@ $mdc-ripple-common-styles-emitted_: false !default;
   }
 
   // Foreground ripple styles
+
```

Comment:
Is this blank line intentional?

Reply:
It matches the one after the "Background ripple styles" comment.